src/InputField.js

```diff
@@ -88,7 +88,7 @@ class InputField extends React.Component {
         if (child.type === Icon) {
           if (inputPosition < 0) {
             BeforeIcon = child
-            iconAdjustment = child.props.size - Icon.defaultProps.size
+            iconAdjustment = child.props.size - 24
```

Comment:
this calculation is off if it's anything besides 24 (current default prop). So it shouldn't refer to a dynamic variable if it only works with one value